ais/api/views.py

```diff
@@ -328,15 +416,9 @@ def search_view(query):
                 .filter_by(**filters)\
                 .order_by_intersection()\
                 .choose_one()
-                #.limit(1)
-
-            if all_intersections is None:
-                all_intersections = intersections
-            else:
-                all_intersections = all_intersections.union(intersections)
 
-            all_intersections = all_intersections.from_self().order_by_intersection()
-            paginator = QueryPaginator(all_intersections)
+            intersections = intersections.from_self().order_by_intersection()
+            paginator = QueryPaginator(intersections)
```

Comment:
I removed pagination, but I recall including it to standardize the response for different search types